lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:dynflow_steps) do
+      if Sequel::Model.db.database_type == :mysql
```

Comment:
using `@db` might be a bit safer here, as I'm not sure how it is with the global attribute `Sequel::Model.db` when there would be more sequel instances connected to different adapters.